content/blog/gpg-signed-releases/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Verifying GPG signatures for Temurin downloads
+date: "2022-07-27T16:00:00+00:00"
+author: sxa
+description: How to verify GPG signatures of the Temurin artifacts using the Eclipse public key
+tags:
+  - Temurin
+  - Security
+---
+
+With the latest releases from July 2022 (8u342, 11.0.16, 17.0.4 and 18.0.2)
+we provide GPG signatures along with our releases which you can use to
+verify that the downloads you have are genuine and have not been tampered
+with since we built them.  This gives a quick crib sheet of what you need to
+do to verify the downloads.
+
+## What is GPG signing?
+
+## You are already providing SHA256 checksums. Why do I need this?
+
+While going into the details of public key encryption is beyond the scope of
+this article, if there was a man in the middle (MITM) attach that resulted
```

Comment:
```suggestion
this article, if there was a man in the middle attack that resulted
```
Typo, and removing the abbreviation since it is not used again.